docs/maintenance-plan.md

```diff
@@ -0,0 +1,189 @@
+# Maintenance plan
+
+This document outlines every piece of the platform that need upgrading and
+updating.
+
+## Lagoon
+
+Update interval: 1-4 months
+Release URL: <https://github.com/uselagoon/lagoon/releases>
+Upgrade Docs URL:
+Expect downtime: Highly likely
+Time to do: Unknown at the time of writing (most likely a day the first time)
+Runbook: [Upgrading Lagoon](./runbooks/upgrading-lagoon.md)
+Notes:
+
+## AKS
+
+Update interval: monthly
```

Comment:
Are we updating AKS monthly?? That seems very very often. Kubernetes releases every 3-4 months, so a cadence around every quarter sohuld be plenty?

Reply:
We should update it montly for a time, until we are up to date. Then we should change it to once every 3 months